crates/test-utils/Cargo.toml

```diff
@@ -34,6 +34,13 @@ tracing-subscriber = { workspace = true, features = ["env-filter"] }
 walkdir.workspace = true
 rand.workspace = true
 snapbox = { version = "0.6.9", features = ["json"] }
+tokio.workspace = true
+
+# zk
+zksync_types.workspace = true
+era_test_node.workspace = true
+jsonrpc-core = { git = "https://github.com/matter-labs/jsonrpc.git", branch = "master" }
+jsonrpc-http-server = { git = "https://github.com/matter-labs/jsonrpc.git", branch = "master" }
```

Comment:
are we using these 2?

Reply:
https://github.com/matter-labs/foundry-zksync/pull/516/files#diff-8f41dcf6a86a9311cf06e14d0fd21030694379f370d9261bbb45f837a706f362R171-R175
and `use jsonrpc_core::IOHandler;` in the same file